libnavigation-core/src/main/java/com/mapbox/navigation/core/NavigationVersionSwitchObserver.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.navigation.core
+
+import com.mapbox.navigation.base.options.RoutingTilesOptions
+
+/**
+ *  An interface which enables listening to navigation tiles version switch.
+ *  Navigator might be switched to a fallback tiles version when there are no enough tiles of the
+ *  current version to navigate on. It might happen when network is not available and tiles can't
+ *  be loaded. When connection is restored, navigator will switch back to the target version
+ *  specified in [RoutingTilesOptions] (the latest available version if no version is specified).
+ *  To create fallback tiles version, tiles should be loaded view the [TileStore].
```

Comment:
```suggestion
 *  
 * **To create additional fallback tiles versions, use the [TileStore] to create and download offline regions.** Otherwise, fallback candidates will only be the tiles versions from the ambient cache.
```

Reply:
done